text/0055-remove-unspecified.md

```diff
@@ -0,0 +1,220 @@
+# Explicit Unspecified Entities
+
+## Related issues and PRs
+
+- Related RFCs: [RFC35](https://github.com/cedar-policy/rfcs/pull/35), [RFC47](https://github.com/cedar-policy/rfcs/pull/47)
+- Implementation PR(s):
+
+## Timeline
+
+- Started: 2024-02-27
+
+## Summary
+
+Cedar currently supports _unspecified entities_, which are entities of a special, unique type that have no attributes, and are not ancestors or descendants of any other entity in the store. Unspecified entities are intended to act as placeholders for entities that don’t influence authorization (see examples below).
+
+The current implementation of unspecified entities results in several corner cases in our code (especially the validator), which have been difficult to maintain (see discussion on [this recent PR](https://github.com/cedar-policy/cedar/pull/603)). The concept of unspecified entities is (in our opinion) confusing for users, and will only become more confusing once we stabilize the partial evaluation experimental feature, which allows “unknown” entities that are subtly different from “unspecified” entities.
+
+This RFC proposes to change the terminology surrounding unspecified entities, instead calling them "default" entities.
+It also proposes to give an explicit name to the unspecified entity type (`__cedar::Default`) and to require this name to be used in requests and schemas that use this type.
+
+## Basic example
+
```

Comment:
Nit: This section is basically the full design, not just a basic example. It might be easier to read/understand this RFC if this section were much shorter, and the details around the design were move further below.

Reply:
Yes it got inflated over time 🙂 Will clean up tomorrow.